docs/coordinates/index.rst

```diff
@@ -566,3 +566,35 @@ Reference/API
 =============
 
 .. automodapi:: astropy.coordinates
+    :skip: ICRS
```

Comment:
This obviously isn't great (it's the whole contents of the `__all__`). I don't think it substantially detracts from the documentation of this page (although maybe it needs a note to say things aren't listed here?). Ideally of course we would have a way of excluding objects on namespace, i.e. do something like `:exclude-namespace: astropy.coordinates.builtin_frames` but :man_shrugging:

Reply:
As long as we add a line that says "go here for frame classes" or something, I think this is acceptable if it is the only way we can unpin Sphinx max version.

Reply:
The classes are listed in the block above, so it's not that hard to spot but I am happy to add a line.

Reply:
Maybe good to see if we can change the transformgraph view to not link to classes - that seems to be the problem (and even now we cannot click on it anyway)

Reply:
Is there actually a problem, other than the links inside the graph not working? (I am not sure they did to begin with)